Source: Upstream
Upstream: Seems to have been reverted in r970127
Reason: akonadi/tray/ needs to include /usr/include/knotificationitem-1, otherwise the build fails:

[ 26%] Building CXX object akonadi/tray/CMakeFiles/akonaditray.dir/main.o                                                                                                                           [0,11/1958]
cd /var/tmp/paludis/build/kde-kdepim-4.2.87/work/build/akonadi/tray && /usr/lib64/ccache/bin/x86_64-pc-linux-gnu-g++   -D_BSD_SOURCE -D_XOPEN_SOURCE=500 -D_BSD_SOURCE -DQT_NO_STL -DQT_NO_CAST_TO_ASCII -D_REE
NTRANT -DKDE_DEPRECATED_WARNINGS -march=core2 -pipe -O2 -ggdb -D__INGMAR_CXXFLAGS__ -DNDEBUG -DQT_NO_DEBUG -Wnon-virtual-dtor -Wno-long-long -ansi -Wundef -Wcast-align -Wchar-subscripts -Wall -W -Wpointer-ar
ith -Wformat-security -fno-exceptions -fno-check-new -fno-common -Woverloaded-virtual -fno-threadsafe-statics -fvisibility=hidden -fvisibility-inlines-hidden -fexceptions -I/var/tmp/paludis/build/kde-kdepim-
4.2.87/work/build/akonadi/tray -I/var/tmp/paludis/build/kde-kdepim-4.2.87/work/kdepim-4.2.87svn969966/akonadi/tray -I/var/tmp/paludis/build/kde-kdepim-4.2.87/work/kdepim-4.2.87svn969966 -I/var/tmp/paludis/bu
ild/kde-kdepim-4.2.87/work/build -I/var/tmp/paludis/build/kde-kdepim-4.2.87/work/kdepim-4.2.87svn969966/libkdepim -I/usr/include/KDE -I/usr/include/qt4/KDE -I/usr/include/qt4 -I/usr/include/qt4/phonon -I/usr
/include/qt4/QtXmlPatterns -I/usr/include/qt4/QtWebKit -I/usr/include/qt4/QtHelp -I/usr/include/qt4/QtAssistant -I/usr/include/qt4/QtDBus -I/usr/include/qt4/QtTest -I/usr/include/qt4/QtUiTools -I/usr/include
/qt4/QtScript -I/usr/include/qt4/QtSvg -I/usr/include/qt4/QtXml -I/usr/include/qt4/QtSql -I/usr/include/qt4/QtOpenGL -I/usr/include/qt4/QtNetwork -I/usr/include/qt4/QtDesigner -I/usr/include/qt4/Qt3Support -
I/usr/include/qt4/QtGui -I/usr/include/qt4/QtCore -I/usr/include/qt4/Qt -I/usr/share/qt4/mkspecs/default   -D_GNU_SOURCE -D_LARGEFILE64_SOURCE -o CMakeFiles/akonaditray.dir/main.o -c /var/tmp/paludis/build/k
de-kdepim-4.2.87/work/kdepim-4.2.87svn969966/akonadi/tray/main.cpp
In file included from /var/tmp/paludis/build/kde-kdepim-4.2.87/work/kdepim-4.2.87svn969966/akonadi/tray/main.cpp:27:
/var/tmp/paludis/build/kde-kdepim-4.2.87/work/kdepim-4.2.87svn969966/akonadi/tray/dock.h:24:31: error: knotificationitem.h: No such file or directory
In file included from /var/tmp/paludis/build/kde-kdepim-4.2.87/work/kdepim-4.2.87svn969966/akonadi/tray/main.cpp:27:
/var/tmp/paludis/build/kde-kdepim-4.2.87/work/kdepim-4.2.87svn969966/akonadi/tray/dock.h:32: error: 'Experimental' is not a namespace-name
/var/tmp/paludis/build/kde-kdepim-4.2.87/work/kdepim-4.2.87svn969966/akonadi/tray/dock.h:32: error: expected namespace-name before ';' token
/var/tmp/paludis/build/kde-kdepim-4.2.87/work/kdepim-4.2.87svn969966/akonadi/tray/dock.h:44: error: expected class-name before '{' token
/var/tmp/paludis/build/kde-kdepim-4.2.87/work/kdepim-4.2.87svn969966/akonadi/tray/dock.h:44: warning: 'class Dock' has virtual functions and accessible non-virtual destructor
make[2]: *** [akonadi/tray/CMakeFiles/akonaditray.dir/main.o] Error 1


From cc378157c50780e746af467dd0be356741c8bd16 Mon Sep 17 00:00:00 2001
From: coles <coles@283d02a7-25f6-0310-bc7c-ecb5cbfe19da>
Date: Tue, 19 May 2009 14:44:50 +0000
Subject: [PATCH] Changed KNotificationItem variable names to match the current values.

git-svn-id: svn://anonsvn.kde.org/home/kde/trunk/KDE/kdepim@970119 283d02a7-25f6-0310-bc7c-ecb5cbfe19da
---
 akonadi/tray/CMakeLists.txt |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/akonadi/tray/CMakeLists.txt b/akonadi/tray/CMakeLists.txt
index 6a4c688..ec46e94 100644
--- a/akonadi/tray/CMakeLists.txt
+++ b/akonadi/tray/CMakeLists.txt
@@ -1,6 +1,6 @@
 INCLUDE_DIRECTORIES( ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR}
                      ${QT_INCLUDE_DIR} ${QT_QTDBUS_INCLUDE_DIR} 
-                     ${LIBKNOTIFICATIONITEM-1_INCLUDE_DIR} )
+                     ${KNOTIFICATIONITEM_INCLUDE_DIR} )
 
 SET(tray_sources main.cpp dock.cpp backup.cpp backupassistant.cpp restore.cpp restoreassistant.cpp global.cpp)
 
@@ -9,7 +9,7 @@ qt4_generate_dbus_interface( ${CMAKE_CURRENT_SOURCE_DIR}/dock.h
 qt4_add_dbus_adaptor( tray_sources org.freedesktop.akonaditray.xml dock.h Dock )
 
 KDE4_ADD_EXECUTABLE(akonaditray ${tray_sources})
-TARGET_LINK_LIBRARIES(akonaditray ${LIBKNOTIFICATIONITEM-1_LIBRARIES} ${KDE4_KIO_LIBS} ${KDE4_KDEUI_LIBS} ${AKONADI_COMMON_LIBRARIES} ${KDEPIMLIBS_AKONADI_LIBS})
+TARGET_LINK_LIBRARIES(akonaditray ${KNOTIFICATIONITEM_LIBRARIES} ${KDE4_KIO_LIBS} ${KDE4_KDEUI_LIBS} ${AKONADI_COMMON_LIBRARIES} ${KDEPIMLIBS_AKONADI_LIBS})
 
 INSTALL(TARGETS akonaditray ${INSTALL_TARGETS_DEFAULT_ARGS})
 INSTALL(FILES  akonaditray.desktop DESTINATION ${XDG_APPS_INSTALL_DIR} )
-- 
1.6.3.1

From 87b4f194ab406d9d9f61ba9c7fe88ed92627fbf7 Mon Sep 17 00:00:00 2001
From: coles <coles@283d02a7-25f6-0310-bc7c-ecb5cbfe19da>
Date: Tue, 19 May 2009 14:40:12 +0000
Subject: [PATCH] Fixed for build with current KNotificationItem variables.

git-svn-id: svn://anonsvn.kde.org/home/kde/trunk/KDE/kdepim@970114 283d02a7-25f6-0310-bc7c-ecb5cbfe19da
---
 korganizer/korgac/CMakeLists.txt |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/korganizer/korgac/CMakeLists.txt b/korganizer/korgac/CMakeLists.txt
index 6204967..cf2495c 100644
--- a/korganizer/korgac/CMakeLists.txt
+++ b/korganizer/korgac/CMakeLists.txt
@@ -3,7 +3,7 @@ add_definitions(-DKDE_DEFAULT_DEBUG_AREA=5890)
 
 add_subdirectory( pixmaps ) 
 add_subdirectory( tests ) 
-include_directories( ${CMAKE_SOURCE_DIR}/korganizer ${LIBKNOTIFICATIONITEM-1_INCLUDE_DIR} )
+include_directories( ${CMAKE_SOURCE_DIR}/korganizer ${KNOTIFICATIONITEM_INCLUDE_DIR} )
 
 
 ########### next target ###############
@@ -26,7 +26,7 @@ kde4_add_app_icon(korgac_SRCS "${KDE4_ICON_DIR}/oxygen/*/apps/korgac.png")
 
 kde4_add_executable(korgac ${korgac_SRCS})
 
-target_link_libraries(korgac ${LIBKNOTIFICATIONITEM-1_LIBRARIES} ${KDE4_KDEUI_LIBS} ${KDE4_PHONON_LIBS} ${KDEPIMLIBS_KCAL_LIBS} korganizer_eventviewer)
+target_link_libraries(korgac ${KNOTIFICATIONITEM_LIBRARIES} ${KDE4_KDEUI_LIBS} ${KDE4_PHONON_LIBS} ${KDEPIMLIBS_KCAL_LIBS} korganizer_eventviewer)
 
 install(TARGETS korgac  ${INSTALL_TARGETS_DEFAULT_ARGS} )
 
-- 
1.6.3.1

